src/app/api/google/google.api.ts

```diff
@@ -0,0 +1,26 @@
+import axios from "axios"
+const googleApiKey = process.env.NEXT_PUBLIC_GOOGLE_API_KEY || undefined;
```

Comment:
`|| undefined` no needed 

At any case can do `|| ''` to guarantee the type is string